test/framework/e2e.go

```diff
@@ -149,7 +155,12 @@ func (e *E2ETest) UpgradeCluster(opts ...E2ETestOpt) {
 		opt(e)
 	}
 	e.buildClusterConfigFile()
-	e.RunEKSA("anywhere", "upgrade", "cluster", "-f", e.ClusterConfigLocation, "-v", "4")
+
+	upgradeClusterArgs := []string{"anywhere", "upgrade", "cluster", "-f", e.ClusterConfigLocation, "-v", "4"}
+	if _, err := os.Stat(defaultBundleReleaseManifestFile); err == nil {
```

Comment:
nit: Instead of looking for the file, shall we set bundlesOverride as an env variable during test run? 
Like we do for job ID here
[L97](https://github.com/aws/eks-anywhere/blob/main/internal/test/e2e/setup.go#L97)
Since we are passing it all the way in we might as well set it in the test env.

Reply:
Sure I can set that. That's better